engine/apps/api/serializers/alert_group.py

```diff
@@ -92,10 +95,33 @@ def get_render_for_web(self, obj):
         if not obj.last_alert:
             return {}
 
-        return AlertGroupWebRenderer(obj, obj.last_alert).render()
+        web_templates_modified_at = obj.channel.web_templates_modified_at
```

Comment:
Shouldn't this be on `AlertGroupListSerializer.get_render_for_web` as well so the list view is cached? I believe `ShortAlertGroupSerializer` is only used to render the root alert group and dependant alert groups.
Probably this caching stuff can be moved to a separate function that's used both on `AlertGroupListSerializer` and `ShortAlertGroupSerializer`?

Reply:
yep, thank you, I want to use this approach in all such serializer methods after I confirm it is working on dev cluster

Reply:
Ignore this comment, it's already on `AlertGroupListSerializer`, it's just GitHub merging diff lines 🙂